### PR TITLE
Add TheiaCon 2021 Website

### PR DESCRIPTION
**Pull request by linkfang**
Fixes [issue222](https://github.com/EclipseFdn/events.eclipse.org/issues/222)

### CODE REVIEW

File: content/2021/theiacon/_index.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+logo: 'images/theiacon-logo.svg'
```
Comment (autumnfound) [inline content/2021/theiacon/_index.md:2]: this should be images/logos/theiacon-logo.svg

Comment (linkfang) [reply content/2021/theiacon/_index.md:2]: Ah, sorry, that's something I forgot to change while trying different things (not the root cause of the issue). 
But thanks a lot for find out the root cause on slack - "2021/theiacon".

**Comment by ericpoirier** [comment]
@linkfang are we still waiting for a +1 here?

**Comment by linkfang** [comment]
> @linkfang are we still waiting for a +1 here?

I think what we have are good. But we are waiting for the agenda.

**Comment by ericpoirier** [comment]
> > @linkfang are we still waiting for a +1 here?
> 
> I think what we have are good. But we are waiting for the agenda.

ok sounds good, just checking if it was ready to merge or not because I saw Mehrnas comment on the issue saying that it looked good.

**Comment by linkfang** [comment]
Hailley requested to push live without agenda section. It will be added next week.
https://github.com/EclipseFdn/events.eclipse.org/issues/222#issuecomment-933552981

**Pull request merged**